prime-router/src/main/kotlin/azure/CheckFunction.kt

```diff
@@ -58,6 +64,7 @@ class CheckFunction : Logging {
 
     /**
      * Checks a single remote connection
+     * Deprecated, I mean like no-one uses this.
```

Comment:
Yeah, would be nice if we could actually use the Deprecated decorator and have the checks pass.